src/components/frame/v5/pages/BalancePage/partials/BalanceTable.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react';
+import Button from '~v5/shared/Button';
+import TableHead from './TableHead';
+import TableItem from './TableItem';
+import { formatText } from '~utils/intl';
+import { useColonyContext } from '~hooks';
+import { BalaceTableProps } from '../types';
+
+const displayName = 'v5.pages.BalancePage.partials.BalaceTable';
+
+const BalaceTable: FC<BalaceTableProps> = ({
+  data,
+  isSorted,
+  onBalanceSort,
+}) => {
+  const { colony } = useColonyContext();
+  const { balances, nativeToken, status } = colony || {};
+  const { nativeToken: nativeTokenStatus } = status || {};
+  const onAddFunds = () => {}; // @TODO: open modal
+
+  return (
+    <>
+      <div className="pb-4">
+        <div className="flex sm:justify-between sm:items-center sm:flex-row flex-col">
+          <div className="flex items-center">
+            <h4 className="heading-5 mr-3">
+              {formatText({ id: 'balancePage.table.title' })}
+            </h4>
+          </div>
+          <div className="flex items-center mt-2.5 sm:mt-0">
+            <Button
+              mode="primarySolid"
+              className="ml-2"
+              onClick={onAddFunds}
+              size="small"
+            >
+              {formatText({ id: 'balancePage.table.addFunds' })}
+            </Button>
+          </div>
+        </div>
+      </div>
+      <TableHead onClick={onBalanceSort} isSorted={isSorted} />
+      {data && (
+        <div className="px-4 border border-gray-200 rounded-b-lg">
```

Comment:
We talked about the fact that for now we can create tables this way because it will be much faster, and when there is more time, functionalities will be added to the TableBase component, such as pagination or filtering. Then this balance table and others such as the verified table will be replaced with the TableBase component.

Reply:
fixed